### ISSUE TITLE
Accessibility - Submitted Screen - Why is There a Message after the Buttons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#67]
Why for the not anonymous incidents submit screen is there a message after the two buttons? Someone using TalkBack could miss them if they select one of the buttons before the text. The text seems important, so it should be before the buttons.
